doc-content/enterprise-only/integration/sso-kie-server-token-proc.adoc

```diff
@@ -1,6 +1,6 @@
 [id='sso-kie-server-token-proc']
 = {KIE_SERVER} token-based authentication
-You can also use token-based authentication for communication between {PRODUCT} and {KIE_SERVER}. You can use the complete token as a system property of your application server, instead of the user name and password, for your applications. However, you must ensure that the token will not expire while the applications are interacting because the token is not automatically refreshed. To get the token, see <<sso-token-auth-proc>>.
+You can also use token-based authentication for communication between {PRODUCT} and {KIE_SERVER}. You can use the complete token as a system property of your application server, rather than the user name and password, for your applications. However, you must ensure that the token will not expire while the applications are interacting because the token is not automatically refreshed. To get the token, see <<sso-token-auth-proc>>.
```

Comment:
I tend to favor "instead of" over "rather than", from a Global English perspective. It's more immediately clear, imo. Your call though.

Reply:
Thanks, the change was based on a section of the IBM SG. I reread this and agree that "instead of" is better. Will revert this change from "rather than" to "instead of".